evalml/pipelines/components/estimators/classifiers/lightgbm_classifier.py

```diff
@@ -69,42 +73,43 @@ def __init__(self, boosting_type="gbdt", learning_rate=0.1, n_estimators=100, ma
                          random_state=random_seed)
 
     def _encode_categories(self, X, fit=False):
-        X2 = pd.DataFrame(copy.copy(X))
-        # encode each categorical feature as an integer
-        X2.columns = np.arange(len(X2.columns))
+        """Encodes each categorical feature using ordinal encoding."""
+        X_encoded = _convert_to_woodwork_structure(X)
+        X_encoded = _rename_column_names_to_numeric(X_encoded)
+        cat_cols = list(X_encoded.select('category').columns)
+        X_encoded = _convert_woodwork_types_wrapper(X_encoded.to_dataframe())
         # necessary to wipe out column names in case any names contain symbols ([, ], <) which LightGBM cannot properly handle
```

Comment:
Should this comment move too? @bchen1116

Reply:
yeah above line 78 would be good, but it can also be removed

Reply:
Removing, since the docstring for `_rename_column_names_to_numeric` explains :)